INSTALL.md

```diff
@@ -23,11 +23,12 @@ fedora | libatomic
 
 ## macOS
 
-Pony can be installed as a Homebrew package.
+Prebuilt macOS packages are available via [ponyup](https://github.com/ponylang/ponyup). You can install nightly builds as well as official releases using ponyup.
```

Comment:
I think we should remove the work "official", since it implies that the nightly releases are not official

Reply:
Perhaps replace with the word "tagged".

Reply:
I'm not a fan of "tagged" nor "official".

Maybe:

"You can also install nightly builds using ponyup."